### PR TITLE
Matic

### PR DESCRIPTION
**Pull request by SkyTradeInc**

### CODE REVIEW

File: lib/components/FileInspector.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ class FileInspector {
         return;
       } else {
         console.log(chalk.yellow(`[+] Missing .env file, attempting to create`))
```
Comment (0xigor) [inline lib/components/FileInspector.js:14]: Please change the console log to logger.

File: package.json [hunk 2]
```diff
@@ -16,12 +16,14 @@
     "dotenv": "^8.1.0",
     "ethereum-input-data-decoder": "^0.2.0",
     "express": "^4.17.1",
-    "jest": "^24.9.0",
     "mongoose": "^5.6.12",
     "request": "^2.88.0",
```
Comment (0xigor) [inline package.json:20]: Request library is deprecated. Please remove this is if not needed.

**Comment by 0xigor** [comment]
@SkyTradeInc , I have added a dev branch. Can you please rebase the PR to dev branch instead of master?

**Comment by SkyTradeInc** [comment]
Moved this to merge with Dev instead as recommended by Omkar

**Pull request closed without merging**